### PR TITLE
Updated LOD Additions and Improvements

### PR DESCRIPTION
**Pull request by Superfly-Johnson**
Fixed it more or less, somehow I am 2 commits behind though? Still should merge

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -5366,13 +5366,50 @@ plugins:
...
+    group: *weatherGroup
+    inc:
+      - 'NevadaSkies - Ultimate DLC Edition.esp'
+      - 'NevadaSkies - TTW Edition.esp'
+  - name: 'NevadaSkies - TTW Edition.esp'
+    url: [ 'https://www.nexusmods.com/newvegas/mods/35998' ]
+    group: *weatherGroup
+    inc:
+      - 'NevadaSkies - Basic Edition.esp'
+      - 'NevadaSkies - Ultimate DLC Edition.esp'
```
Comment (MacSplody) [inline masterlist.yaml]: This might have been copied over on a rebase or merge.
It doesn't look related to the other changes.

```yaml
  - name: 'NevadaSkies - Ultimate DLC Edition.esp'
    url: [ 'https://www.nexusmods.com/newvegas/mods/35998' ]
    group: *weatherGroup
    inc:
      - 'NevadaSkies - Basic Edition.esp'
      - 'NevadaSkies - TTW Edition.esp'
  - name: 'NevadaSkies - Basic Edition.esp'
    url: [ 'https://www.nexusmods.com/newvegas/mods/35998' ]
    group: *weatherGroup
    inc:
      - 'NevadaSkies - Ultimate DLC Edition.esp'
      - 'NevadaSkies - TTW Edition.esp'
  - name: 'NevadaSkies - TTW Edition.esp'
    url: [ 'https://www.nexusmods.com/newvegas/mods/35998' ]
    group: *weatherGroup
    inc:
      - 'NevadaSkies - Basic Edition.esp'
      - 'NevadaSkies - Ultimate DLC Edition.esp'
```

Comment (Superfly-Johnson) [reply masterlist.yaml]: yeah it's a merge, didn't expect that to somehow end up in it, very weird. I'll have to redo it again :(

Comment (MacSplody) [reply masterlist.yaml]: Well you add a commit removing it, and then I can squash the changes when merging.
The rest of the changes seem fine, though I would generally place 3rd party patches into their own section.

Comment (Superfly-Johnson) [reply masterlist.yaml]: > Well you add a commit removing it, and then I can squash the changes when merging. The rest of the changes seem fine, though I would generally place 3rd party patches into their own section.

Removing it how?

Comment (MacSplody) [inline masterlist.yaml]:
```suggestion
```

Comment (MacSplody) [reply masterlist.yaml]: I removed it using the suggestion.

image

Comment (MacSplody) [reply masterlist.yaml]: And then when I merge, I'll squash and remove the extra commit.

**Review by MacSplody — APPROVED** [review]

**Comment by MacSplody** [comment]
Thanks.
